core/spec.md

```diff
@@ -850,17 +883,16 @@ of the existing entity. Then the existing entity can be deleted.
     removing an entity from a nested xRegistry collection MUST update the
     `modifiedat` value of the parent entity.
   - If present in a write operation request, the following applies:
-    - If the request value is the same as the existing value, then the
-      current date/time MUST be used as its new value.
+    - If the request value is `null` or the same as the existing value, then
+      the current date/time MUST be used as its new value.
     - If the request value is different than the existing value, then the
       request value MUST be used as its new value.
-    - If the request value is `null` then the current date/time MUST be used
-      as the new value.
   - When absent in a write operation request, it MUST be set to the current
     date/time.
-  - In cases where the `modifiedat` attribute is set to the current date/time
-    on multiple entities within the same operation, the same value MUST be
-    applied to all of the entities.
+  - During the processing of a single request, all entities that have their
```

Comment:
Same question as earlier about how this could be enforced.

Reply:
same change as above